### ISSUE TITLE
error when using process or bidsify

### ISSUE THREAD

**Issue by Remi-Gau** [nbeliy/bidsme#2]
I am getting an error I am having a hard time to parse when trying to process or bidsify.

Preparing the data set and setting the bidsmap was OK, but then I want to turn my 3D func into 4D using process and a plugin and I get this error.

I actually get the same error on a dru run with bidsify.

Also this error only happens when it starts processing the func files not before.

```
bidsme(148) - ERROR /home/remi/github/bidsme/bidsme/bidsme.py(122) in <module>: 
bidsme(148) - ERROR /home/remi/github/bidsme/bidsme/bidsify.py(383) in bidsify: 
bidsme(148) - ERROR /home/remi/github/bidsme/bidsme/bidsify.py(97) in coin: 
bidsme(148) - ERROR /home/remi/github/bidsme/bidsme/Modules/base.py(1112) in getBidsname: 
bidsme(148) - ERROR /home/remi/github/bidsme/bidsme/tools/tools.py(74) in cleanup_value: 
bidsme(149) - ERROR 1:AttributeError: 'int' object has no attribute 'strip'
```

I figured you might be faster at tracking it down.

**Comment by nbeliy** [comment]
Can you attach the bidsmap entry corresponding to scan, which provokes this error? 

My guess, one of the values in the `bids` section is a number. But I'm surprised why bidsmap didn't crashed with same error.

**Comment by nbeliy** [comment]
Made a branch "fixes" that fix this issue and issue #5 
Can you check if it works for you?

For this issue, if one of values in bids section contains non-string, a warning should appear, and this value is forcefully casted to string. Warning is needed because it is unclear how interpret float and negative numbers as values of bids entities.

**Comment by Remi-Gau** [comment]
yup that fixed it.

awesome: thanks.

now I have to create perfect that plugin to concatenate 3D into 4D. :wink:

**Comment by nbeliy** [comment]
fixed in d79916288fff1df
